planning/behavior_velocity_planner/include/utilization/util.hpp

```diff
@@ -137,6 +137,14 @@ double calcJudgeLineDistWithJerkLimit(
   const double velocity, const double acceleration, const double max_stop_acceleration,
   const double max_stop_jerk, const double delay_response_time);
 
+double calcMaxSlowDownVelocityFromDistanceToTarget(
+  const double max_slowdown_jerk, const double max_slowdown_accel, const double current_accel,
+  const double current_velocity, const double distance_to_target);
+
+double findReachTime(
```

Comment:
This is utility function. So, args of function should be more easy to read.
(jerk, accel, velocity...)